gulpfile.js

```diff
@@ -87,9 +87,9 @@ gulp.task('javascript', function () {
   if (!process.env.DS_ENV) {
     if (!process.env.TRAVIS_BRANCH
```

Comment:
The `!process.env.TRAVIS_BRANCH` bit is what will happen when developing locally, so this change would mean that, by default, local development would point at staging.  Just to confirm, is that what we want?  @jflasher